sunpy-dev-env.yml

```diff
@@ -35,6 +35,7 @@ dependencies:
   # Testing
   - hypothesis
   - pytest
+  - pytest-arraydiff
```

Comment:
This also needs to be enabled by passing `--arraydiff` to the pytest command.

Reply:
I'm adding it to the environment file in anticipation (and it doesn't matter because it's already installed anyway due to `pytest-astropy`).  Actually enabling it our tox commands requires me to fix that one test.